scripts/snapshot.mjs

```diff
@@ -15,7 +15,8 @@ const featureFlags = [
   'vitest',
   'cypress',
   'playwright',
-  'nightwatch'
+  'nightwatch',
+  'vue-i18n'
```

Comment:
I think it should be removed from here because it's going to generate too many snapshots